src/common/src/Modal/ModalContent/ModalContent.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import ModalContent from './ModalContent';
+
+describe('<ModalContent />', () => {
+	it('should remove the margin from the first and last element', () => {
```

Comment:
Not sure what you want to test here. Don't see any dynamic input/values which might change the margins. I would opt for a more explicit approach. Probably the margins are changing based on a boolean, but I don't see that logic anywhere